RecoMuon/MuonIdentification/BuildFile.xml

```diff
@@ -26,11 +26,19 @@
 <use   name="FWCore/Framework"/>
 <use   name="FWCore/MessageLogger"/>
 <use   name="FWCore/ParameterSet"/>
+<use   name="FWCore/Utilities"/>
+
 <use   name="Geometry/CommonDetUnit"/>
 <use   name="Geometry/CSCGeometry"/>
+<use   name="Geometry/GEMGeometry"/>
 <use   name="Geometry/Records"/>
 <use   name="SimDataFormats/Track"/>
 <use   name="SimDataFormats/TrackingHit"/>
+<use   name="SimDataFormats/Associations"/>
```

Comment:
Are these Sim dependencies really needed?
We should avoid this for Reco* packages